src/contracts/core/AllocationManagerStorage.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.27;
+
+import "../interfaces/IAVSDirectory.sol";
+import "../interfaces/IAllocationManager.sol";
+import "../interfaces/IDelegationManager.sol";
+import {Snapshots} from "../libraries/Snapshots.sol";
+
+abstract contract AllocationManagerStorage is IAllocationManager {
+    using Snapshots for Snapshots.History;
+
+    /// @notice The EIP-712 typehash for the contract's domain
+    bytes32 public constant DOMAIN_TYPEHASH =
+        keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
+
+    /// @notice The EIP-712 typehash for the `MagnitudeAdjustments` struct used by the contract
+    bytes32 public constant MAGNITUDE_ADJUSTMENT_TYPEHASH = keccak256(
+        "MagnitudeAdjustments(address operator,MagnitudeAdjustment(address strategy, OperatorSet(address avs, uint32 operatorSetId)[], uint64[] magnitudeDiffs)[],bytes32 salt,uint256 expiry)"
```

Comment:
very minor nit but I think you're not supposed to have any spaces after commas in this definition (as-is this is inconsistent with these)

Reply:
removed typehash